<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCTDialog</class>
 <widget class="QDialog" name="DCTDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCT encoding</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Encoding mode &amp;&amp; associated parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="truncButton">
        <property name="text">
         <string>Coefficients truncation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="truncLimitLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Truncation limit : </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="truncLimitBox">
          <property name="maximum">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="bitButton">
        <property name="text">
         <string>Bit allocation matrice</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="initBitLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Initial number of bits : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="initBitBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="slopeLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Slope value : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="slopeBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>8.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DCTDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DCTDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>truncButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>truncLimitLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>initBitLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>initBitBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>slopeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>truncButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>truncLimitBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>slopeBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
